numpy-stubs/linalg.py

```diff
@@ -0,0 +1,7 @@
+from typing import Tuple
+
+from numpy import ndarray, _Int, float64, _DType
+
+
+def slogdet(x: ndarray[_DType]) -> Tuple[_Int, ndarray[float64]]:
```

Comment:
```suggestion
def slogdet(x: ndarray[_DType]) -> Tuple[int, ndarray[float64]]:
```

same here. is `int` the right type? or is it `np.integer`.

Reply:
```
a, b = np.linalg.slogdet(np.eye(500) * 0.1)
type(a) # <class 'numpy.float64'>
```
Looks like it actually returns np.float64